catalog/transformer/catalog-transformer-pptx/src/main/java/ddf/catalog/transformer/input/pptx/PptxInputTransformer.java

```diff
@@ -165,6 +165,8 @@ private void extractThumbnail(Metacard metacard, InputStream input) throws IOExc
         LOGGER.debug("Cannot transform old style (OLE2) ppt : id = {}", metacard.getId());
       }
 
+    } catch (NoClassDefFoundError e) {
+      LOGGER.warn("Cannot extract thumbnail: ", e);
```

Comment:
Shouldn't this throw an IOException?

Reply:
If we threw an IOException here would that get caught in MetacardFactory.generateMetacard() ? I think that would lead to a debug message of `"could not create metacard"` and the stacktrace being called, although not necessarily used.

The way this change currently works is that the pptx transformer is creating the metacard, albeit without the thumbnail.

Reply:
Good point. I guess we don't need to throw the exception here.